src/css/profile/mobile/common/dropdownmenu.less

```diff
@@ -213,20 +213,23 @@
 		margin: 0;
 		background-color: var(--dropdown-menu-options-background);
 		border-radius: 26 * @px_base;
-		border: 1px solid var(--dropdown-menu-options-border);
+		box-shadow: inset 0 -3 * @px_base 0 0 rgba(0, 0, 0, 0.35);
```

Comment:
add variable for color

Reply:
This color is constant for both light and dark theme. Should we have a variable for this?

Reply:
yes. Common colors define in "base.less"
`@<variable_name> = fade(@_black, 0.35)`

Reply:
Ok, added.